charts/argo-cd/values.yaml

```diff
@@ -123,26 +123,29 @@ controller:
   replicas: 1
 
   ## Application controller commandline flags
```

Comment:
```suggestion
  # -- DEPRECATED Application controller commandline flags
```

Reply:
I would like to keep this for refactoring and drop all `extraEnv` and `extraArgs` and keep just `args` and `env`.

Reply:
Then we need to add something different. Currently the key is undocumented in README (line 404):
```
| controller.args | object | `{}` | |
```

Reply:
OK marked as deprecated and then we will change the meaning during the recatoring.